### PR TITLE
[WV-334] fixed dict values for candidates view TEAM REVIEW

### PR DESCRIPTION
**Pull request by lredmo1**
Created template values for 
- district name 
- contest office name 
- vote usa office id
- vote usa politician id
- 
Removed lime green background color

### CODE REVIEW

File: candidate/views_admin.py [hunk 1]
```diff
@@ -1830,6 +1830,8 @@ def candidate_edit_view(request, candidate_id=0, candidate_we_vote_id=""):
     candidate_twitter_handle3 = request.GET.get('candidate_twitter_handle3', False)
     candidate_url = request.GET.get('candidate_url', False)
     candidate_contact_form_url = request.GET.get('candidate_contact_form_url', False)
+    contest_office_name = request.GET.get('contest_office_name', False)
+    district_name = request.GET.get('district_name', False)
```
Comment (lredmo1) [inline candidate/views_admin.py:1834]: Wasn't sure if these should be `False` or `""`

File: candidate/views_admin.py [hunk 2]
```diff
@@ -1853,7 +1855,9 @@ def candidate_edit_view(request, candidate_id=0, candidate_we_vote_id=""):
     withdrawal_date = request.GET.get('withdrawal_date', False)
     withdrawn_from_election = positive_value_exists(request.GET.get('withdrawn_from_election', False))
     do_not_display_on_ballot = positive_value_exists(request.GET.get('do_not_display_on_ballot', False))
-
+    vote_usa_office_id = request.GET.get('vote_usa_office_id', False)
+    vote_usa_politician_id = request.GET.get('vote_usa_politician_id', False)
```
Comment (lredmo1) [inline candidate/views_admin.py:1859]: Same question here re: `False` value

File: candidate/views_admin.py [hunk 3]
```diff
@@ -2000,61 +2004,61 @@ def candidate_edit_view(request, candidate_id=0, candidate_we_vote_id=""):
             'ballotpedia_candidate_id':         ballotpedia_candidate_id,
             'ballotpedia_candidate_id_dict':              
             {
-                'label': 'Candidate Name (from Ballotpedia)',
-                'id': 'ballotpedia_candidate_id_id',
-                'name': 'ballotpedia_candidate_id',
-                'value': ballotpedia_candidate_id if ballotpedia_candidate_id else candidate_on_stage.ballotpedia_candidate_id
-			},
```
Comment (lredmo1) [inline candidate/views_admin.py:2007]: FYI these are inline with open brackets, just doesn't look like it on the diff for some reason

File: candidate/views_admin.py (hunk 1 above)
Comment (DaleMcGrew) [reply candidate/views_admin.py:1834]: False is correct -- a "False" value is the signal we use to specify that a GET variable wasn't received.

**Comment by DaleMcGrew** [comment]
👍  Thank you @lredmo1!

**Pull request merged**